lib/__tests__/findMatchingFiles-test.js

```diff
@@ -6,103 +6,121 @@ import rimraf from 'rimraf';
 
 import findMatchingFiles from '../findMatchingFiles';
 
-describe('findMatchingFiles', () => {
-  const tmpDir = './tmp-findMatchingFiles';
-  let subject;
-  let word;
-  let existingFiles;
-  let lookupPath;
-
-  beforeEach(() => {
-    fs.mkdirSync(tmpDir);
-    word = 'foo';
-    existingFiles = [];
-    lookupPath = tmpDir;
-
-    subject = () => {
-      existingFiles.forEach((file) => {
-        const fullPath = `${tmpDir}/${file}`;
-        mkdirp.sync(path.dirname(fullPath));
-        fs.closeSync(fs.openSync(fullPath, 'w')); // create empty file
+['node', 'find'].forEach((finder) => {
```

Comment:
Yeah. I wanted to have the same test coverage though, without duplicating things. I think this is okay.